src/main/java/me/xmrvizzy/skyblocker/skyblock/item/PriceInfoTooltip.java

```diff
@@ -128,24 +170,14 @@ public static void onInjectTooltip(ItemStack stack, TooltipContext context, List
                             .formatted(Formatting.LIGHT_PURPLE)
                             .append(new LiteralText(timestamp != null ? timestamp : "").formatted(Formatting.RED)));
                 }
-            } else if (!listString.contains("Obtained") && timestamp != null && SkyblockerConfig.get().general.itemTooltip.enableMuseumDate) {
+            } else if (SkyblockerConfig.get().general.itemTooltip.enableMuseumDate && !listString.contains("Obtained") && timestamp != null) {
                 lines.add(new LiteralText(String.format("%-22s", "Obtained: "))
                         .formatted(Formatting.LIGHT_PURPLE)
                         .append(new LiteralText(timestamp).formatted(Formatting.RED)));
             }
-        } catch (NullPointerException ex) {
-            if ((npcPricesJson == null || bazaarPricesJson == null || oneDayAvgPricesJson == null
-                    || threeDayAvgPricesJson == null || lowestPricesJson == null || isMuseumJson == null)
-                    && !nullMsgSend)  {
-                MinecraftClient.getInstance().player.sendMessage(new TranslatableText("skyblocker.itemTooltip.NullMessage"), false);
-                nullMsgSend = true;
-                skyblocker.scheduler.schedule(() -> nullMsgSend = false, 1200);
-            }
-        } catch (Exception ex) {
-            if (MinecraftClient.getInstance().player == null) {
-                throw new RuntimeException("[Skyblocker] client.player cannot be null!");
-            }
-            MinecraftClient.getInstance().player.sendMessage(new LiteralText(ex.toString()), false);
+
+        } catch (Exception e) {
+            client.player.sendMessage(new LiteralText(e.toString()), false);
```

Comment:
Telling player name of the exception is not a good replacement for stack trace in the logs.

Reply:
I didn't write this part of the code but I'll take a look at it when I have time.